packages/paste-core/core-bundle/src/index.tsx

```diff
@@ -1,4 +1,5 @@
 export * from '@twilio-paste/absolute';
+export * from '@twilio-paste/alert';
```

Comment:
I'm concerned `core` may not work this way anymore, since we're exporting `VARIANTS` from the Alert package, and undoubtedly another component will export Variants too.  Either we prefix all exports with the component name (i.e.: `AlertVariants`) in the component packages, or we manually maintain this file to do the renaming.  Thoughts?

Reply:
Switching all the exports, for sure this is going to be a problem